packages/flutter/lib/src/material/icon.dart

```diff
@@ -65,13 +83,13 @@ class Icon extends StatelessComponent {
       category = parts[0];
       subtype = parts[1];
     }
-    // TODO(eseidel): This clearly isn't correct.  Not sure what would be.
-    // Should we use the ios images on ios?
-    String density = 'drawable-xxhdpi';
+    // TODO(kgiesing): Should selector logic be moved down a level so that
+    // any generic asset can be resolved using consistent contextual logic?
```

Comment:
I don't think we yet have enough information to make an educated decision on this, so we should just pick one and iterate. We'll see what our customers do.